nilearn/regions/tests/test_signal_extraction.py

```diff
@@ -52,26 +52,28 @@ def test__check_shape_and_affine_compatibility_with_dim():
 
     with pytest.warns(None):
         signal_extraction._check_shape_and_affine_compatibility(
-            img1,
-            img2,
-            dim=3)
+            img1, img2, dim=3
+        )
+
 
 @pytest.mark.parametrize(
-    "test_shape,test_affine,msg", 
-    [((2, 3, 5), np.eye(4), _TEST_SHAPE_ERROR_MSG),
-     ((2, 3, 4), 2*np.eye(4), _TEST_AFFINE_ERROR_MSG)
-     ])
-def test__check_shape_and_affine_compatibility_error(test_shape,test_affine,msg):
+    "test_shape,test_affine,msg",
```

Comment:
```suggestion
    "test_shape, test_affine, msg",
```

Reply:
Unsure about that one.
Although I don't prefer it, I think I have almost always seen parameters being separated by just a comma.

If we start doing this I may to recheck previously formatted module.

Also there is no way that I know of to enforce this.

Reply:
Ok I didn't know that. Then no worries, leave it as is